Submodels/MTSET_call.py

```
@@ -1,63 +1,110 @@
+- V0.6: Updated documentation & type hinting. Removed dummy inputs & streamlined execution. 
 """
 
 import subprocess
 import os
 
 class MTSET_call:
+    """
+    Class to handle the interface between MTSET and Python
+    """
 
+    def __init__(self, *args,
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)